style/js/shield_defs.js

```diff
@@ -103,6 +103,9 @@ export function loadShields(shieldImages) {
     },
   };
 
+  shields["US:I:Alternate"] = banneredShield(shields["US:I"], ["ALT"]);
+  shields["US:I:Truck"] = banneredShield(shields["US:I"], ["TRK"]);
```

Comment:
For I-5 Truck at Newhall Pass, Caltrans doesn’t post “TRUCK” banners per se, but it adds “[TRUCK ROUTE](https://www.mapillary.com/app/?pKey=2910481322600458)” or “[ALL TRUCKS](https://www.mapillary.com/app/?pKey=230893205470721)” next to the normal I-5 shield.

The route relations were [retagged](https://www.openstreetmap.org/changeset/115758223) with `network=US:I:Truck` too recently for MapTiler to have picked up the correction yet.

Reply:
This can be tested with my planetiler server, http://vtile.streetferret.com:8080/

Reply:
Interstate Truck:
![image](https://user-images.githubusercontent.com/3254090/154866736-edebc6d5-0959-416e-a04f-d24ebbb4dec5.png)